modules/programs/firefox.nix

```diff
@@ -101,9 +101,14 @@ in
           ]
         '';
         description = ''
-          List of Firefox add-on packages to install. Note, it is
-          necessary to manually enable these extensions inside Firefox
-          after the first installation.
+          List of Firefox add-on packages to install. Some addons are
```

Comment:
I think this is a good change. The only concern I have is that I think the link URLs need to be shown in full so that they become visible in the man page. Actually, I think it may be sufficient to just show the NUR link and say that a list of available add-ons can be found by running

```console
$ nix-env -f '<nixpkgs>' -qaP -A nur.repos.rycee.firefox-addons
```

Reply:
Fixed.